<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registroCompras</class>
 <widget class="QMainWindow" name="registroCompras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(61, 61, 61);
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QFrame" name="frame_9">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>REGISTRO COMPRAS</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>504</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="exit_button">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(170, 170, 0);
		border-radius: 50px;
		color: #fff;
}

QPushButton:hover {
 background-color: rgb(170, 170, 150);

}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>CODIGO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>CATEGORIA:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>MARCA:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>MODELO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>DESCRIPCION:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>PRECIO COMPRA:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>PRECIO VENTA:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>STOCK:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>USUARIO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>TOTAL COMPRA:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>23</number>
               </property>
               <item>
                <widget class="QLineEdit" name="codigo_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="categoria_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="marca_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="modelo_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="descripcion_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="preciocompra_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="precioventa_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="stock_compra"/>
               </item>
               <item>
                <widget class="QLineEdit" name="usuario_compra">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="totalcompra_compra">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(170, 170, 0);
		border-radius: 4px;
		color: #fff;
}

QPushButton:hover {
 background-color: rgb(170, 170, 150);

}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="buscar_button_registro_compra">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>BUSCAR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QFrame" name="frame_8">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <property name="spacing">
                    <number>20</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_12">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>FECHA:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_14">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>HORA:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_16">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>USUARIO:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_7">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="spacing">
                    <number>20</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="fecha_wreg">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>FECHA:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="hora_wreg">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>HORA:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="usuario_wreg">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>41</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>USU</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="codusu_wreg">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>41</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>COD_USU</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="registrar_button_registro_compra">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>REGISTRAR</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
